internal/handlers/pg/msg_listdatabases.go

```diff
@@ -67,7 +67,7 @@ func (h *Handler) MsgListDatabases(ctx context.Context, msg *wire.OpMsg) (*wire.
 			for _, name := range tables {
 				var tableSize int64
 				fullName := pgx.Identifier{databaseName, name}.Sanitize()
-				err = tx.QueryRow(ctx, "SELECT pg_total_relation_size($1)", fullName).Scan(&tableSize)
+				err = tx.QueryRow(ctx, "SELECT COALESCE(pg_total_relation_size($1), 0)", fullName).Scan(&tableSize)
```

Comment:
Somehow this is what happened on CI:

```
  logger.go:130: 2022-08-23T14:12:23.571Z	ERROR	// 127.0.0.1:53930 -> 127.0.0.1:45383 	clientconn/conn.go:438	Response message:
        {
          "Checksum": 0,
          "FlagBits": 0,
          "Sections": [
            {
              "Document": {
                "$k": [
                  "ok",
                  "errmsg",
                  "code",
                  "codeName"
                ],
                "ok": {
                  "$f": 0
                },
                "errmsg": "[pool.go:273 pgdb.(*Pool).InTransaction] [msg_listdatabases.go:72 pg.(*Handler).MsgListDatabases.func1] can't scan into dest[0]: cannot scan null into *int64",
                "code": 1,
                "codeName": "InternalError"
              },
              "Kind": 0
            }
          ]
        }
```

See https://github.com/FerretDB/FerretDB/runs/7975285072?check_suite_focus=true#step:8:152.